src/api/Controllers/BoreholeController.cs

```diff
@@ -149,56 +151,73 @@ private IQueryable<Borehole> GetBoreholesWithIncludes()
-                b.ElevationZ,
-                b.ElevationPrecisionId,
-                b.ReferenceElevation,
-                b.ReferenceElevationTypeId,
-                b.ReferenceElevationPrecisionId,
-                b.HrsId,
-                b.TypeId,
-                b.PurposeId,
-                b.StatusId,
-                b.Remarks,
-                b.TotalDepth,
-                b.DepthPrecisionId,
-                b.TopBedrockFreshMd,
-                b.TopBedrockWeatheredMd,
-                b.HasGroundwater,
-                b.LithologyTopBedrockId,
-                b.ChronostratigraphyTopBedrockId,
-                b.LithostratigraphyTopBedrockId,
+                Id = b.Id,
+                OriginalName = b.OriginalName,
+                ProjectName = b.ProjectName,
+                Name = b.Name,
+                RestrictionId = b.RestrictionId,
+                RestrictionUntil = b.RestrictionUntil,
+                NationalInterest = b.NationalInterest,
+                LocationX = b.LocationX,
+                LocationY = b.LocationY,
+                LocationPrecisionId = b.LocationPrecisionId,
+                ElevationZ = b.ElevationZ,
+                ElevationPrecisionId = b.ElevationPrecisionId,
+                ReferenceElevation = b.ReferenceElevation,
+                ReferenceElevationTypeId = b.ReferenceElevationTypeId,
+                ReferenceElevationPrecisionId = b.ReferenceElevationPrecisionId,
+                HrsId = b.HrsId,
+                TypeId = b.TypeId,
+                PurposeId = b.PurposeId,
+                StatusId = b.StatusId,
+                Remarks = b.Remarks,
+                TotalDepth = b.TotalDepth,
+                DepthPresicionId = b.DepthPrecisionId,
+                TopBedrockFreshMd = b.TopBedrockFreshMd,
+                TopBedrockWeatheredMd = b.TopBedrockWeatheredMd,
+                HasGroundwater = b.HasGroundwater,
+                LithologyTopBedrockId = b.LithologyTopBedrockId,
+                ChronostratigraphyTopBedrockId = b.ChronostratigraphyTopBedrockId,
+                LithostratigraphyTopBedrockId = b.LithostratigraphyTopBedrockId,
             })
             .ToListAsync()
             .ConfigureAwait(false);
 
         if (boreholes.Count == 0) return NotFound("No borehole(s) found for the provided id(s).");
 
+        foreach (var b in boreholes)
+        {
+            var boreholeGeometry = await Context.BoreholeGeometry
+            .AsNoTracking()
+            .Where(g => g.BoreholeId == b.Id)
+            .ToListAsync()
+            .ConfigureAwait(false);
+
+            b.TotalDepthTvd = BoreholeGeometryController.GetTVDIfGeometryExists(b.TotalDepth, boreholeGeometry);
+            b.TopBedrockFreshTvd = BoreholeGeometryController.GetTVDIfGeometryExists(b.TopBedrockFreshMd, boreholeGeometry);
+            b.TopBedrockWeatheredTvd = BoreholeGeometryController.GetTVDIfGeometryExists(b.TopBedrockWeatheredMd, boreholeGeometry);
```

Comment:
Q/REQ: Lässt sich die Methode in die `Helper`-Klasse, wo auch die Methode `GetDepthTVD()` liegt auslagern? So rufen sich die Controller nicht gegenseitig auf. `GetTVDIfGeometryExists` kann zum Beispiel als Extension von `List<BoreholeGeometryElement>` definiert werden.

Wenn man gerade dabei ist und ich den Namen `Helper` nicht gut finde: Im `Helper` hat es einige Extensions auf `List<BoreholeGeometryElement>` oder `BoreholeGeometryElement`, welche in eine eigene Klasse `BoreholeGeometryElementExtensions` ausgelagert werden können. Das macht das Ganze klarer und man packt nicht alles automatisch in diese `Helper`-Klasse 😁. Man muss dann schauen, was noch übrig bleibt und ob man diese Methoden ebenfalls weg bringt oder den Klassennamen treffender umformulieren kann.